x-pack/plugins/translations/translations/zh-CN.json

```
@@ -11323,4 +11323,4 @@
     "xpack.fileUpload.fileParser.errorReadingFile": "读取文件时出错",
     "xpack.fileUpload.fileParser.noFileProvided": "错误，未提供任何文件"
   }
-}
+}
```

I think these are unintentional -- does regenerating them return the newline?

---

Hey @spong 
I regenerated them, but they didn't change. Is it a concern or can I merge?

---

Interesting. Judging by the history it looks like it's bounced back and forth between having a terminating new line and not. Generally speaking [you want the terminating new line](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file), so I would just revert these two files since that's the only changes.

---

```suggestion
}

```

Can try accepting this suggestion from within Github as well too 🙂